### PR TITLE
fix: Update README.md to use @v0.2.0 instead of main

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s):

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,12 +69,12 @@ jobs:
     steps:
       - name: Setup AtlasCLI and create a project
         id: create-project
-        uses: mongodb/atlas-github-action@main
```
Comment (lantoli) [inline README.md:72]: also change line 45

**Review by lantoli — APPROVED** [review]

**Review by lantoli — CHANGES_REQUESTED** [review]
it seems like it doesn't work with latest

**Review by matt-condon — APPROVED** [review]

**Comment by matt-condon** [comment]
Ah ok there's nothing in the docs to suggest that this is supported

File: README.md [hunk 2]
```diff
@@ -69,12 +69,12 @@ jobs:
...
+        uses: mongodb/atlas-github-action@latest
```
Comment (matt-condon) [inline README.md:72]: personally partial to
```suggestion
        uses: mongodb/atlas-github-action@v0
```
Then we can keep the docs up to date for major releases

Comment (andreaangiolillo) [reply README.md]: updated to use v0.2.0

**Review by gssbzn — APPROVED** [review]

**Comment by matt-condon** [comment]
Not a huge fan of manual doc updates that will be required with each release, but I guess we can automate it later

**Review by lantoli — APPROVED** [review]

**Pull request merged**